README.md

```diff
@@ -126,6 +124,47 @@ df2 = @byrow df begin
 end
 ```
 
+## Working with column names programmatically with `cols`
+
+DataFramesMeta provides the special syntax, `cols`, for referring to 
```

Comment:
```suggestion
DataFramesMeta provides the special syntax `cols`, for referring to 
```

Reply:
ths comma is necessary

Reply:
AFAIK it has to be either "the special syntax `cols`" or "a special syntax, `cols`". :-)

Reply:
I know you always correct my writing! but I promise the comma belongs lol, [source](https://opinionator.blogs.nytimes.com/2012/05/21/the-most-comma-mistakes/)

Reply:
Well we're exactly in the "I went to see the movie “Midnight in Paris” with my friend Jessie." case AFAICT! :-D

Reply:
I will change it to get this merged! But this is the *only* syntax with those properties, so we are in the "latest movie" case.

Reply:
But it's not the only "special syntax" in the world, contrary to "Woody Allen’s latest movie"!

Reply:
Okay i see your point! 

With this settled, let's merge this! It will put us ahead of `dplyr` in this respect.